contents/articles/2016-10_yamen_hmd_calibration/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: Layered Telepresence
+date: Thu Oct 20 2016 14:14:11 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Yamen Saraiji have developed a Layered Telepresence system which is a layered-based approach to create a seamless interaction between multiple users from multiple locations to engage in the same conversation...."
+featured_img_thumb: ""
+---
+
+[Yamen Saraiji](http://myamens.com/) have developed a Layered Telepresence system which is a layered-based approach to create a seamless interaction between multiple users from multiple locations to engage in the same conversation.
+
+The system combines multiple locations and arrange them as layers that are presented in the same virtual space. The concept allows users to achieve multiple types of compound presence in both physical and virtual environments.
+
+
+<div class="Feature-video-container-16by9">
+	<iframe class="Feature-video" src="https://player.vimeo.com/video/162990147" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+</div>
+
+Check out [Yamen's SIGGRAPH 2016 poster](http://s2016.siggraph.org/poster_pdfs/Poster_24_-_0074.pdf) 
```

Comment:
Check out Yamen Saraiji's SIGGRAPH 2016 poster - [title of poster](link to poster pdf)